src/Core/src/Platform/iOS/MauiRefreshView.cs

```diff
@@ -61,6 +61,11 @@ public void UpdateContent(IView? content, IMauiContext? mauiContext)
 
 			_contentView?.RemoveFromSuperview();
 
+			if(_contentView is not null)
+			{
+				_refreshControl = new UIRefreshControl();
```

Comment:
Could you share more details? Why do we need to create a `UIRefreshControl` instance here?